### DISCUSSION TITLE
Unable to debug as no symbols have been loaded

### DISCUSSION THREAD

**Discussion by acrigney** [unoplatform/uno#10249] [Q&A]
For the ToDo app I want to test the Uno.WinUI.MSAL authentication. But when I debug the Wasm application VS2022 says that no symbols have been loaded. I am using VS2022 17.3.6 and uno-check is all good.

**Comment by jeromelaban** [comment]
Thanks for the report. Can you provide the browser logs for the running app? Which version of Uno.Wasm.Bootstrap is being used in your app? Can you also post a screenshot of the "not symbols have been loaded"?

**Reply by acrigney** [reply]
Thanks mate, sounds good but I tried to add InitializeLogging(); which is from the [Uno.Extensions.Logging.WebAssembly.Console](https://www.nuget.org/packages/Uno.Extensions.Logging.WebAssembly.Console/) which is included in the ToDo project but as soon as I try and change the App.cs file it looks like the code generation fails.
As I get
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS1061	'App' does not contain a definition for 'InitializeComponent' and no accessible extension method 'InitializeComponent' accepting a first argument of type 'App' could be found (are you missing a using directive or an assembly reference?)	ToDo.Wasm, ToDo.Windows.Desktop	C:\Projects\uno.todo-main\uno.todo-main\src\ToDo.UI\App.xaml.cs	16	Active

And
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0103	The name 'InitializeLogging' does not exist in the current context	ToDo.Wasm, ToDo.Windows.Desktop	C:\Projects\uno.todo-main\uno.todo-main\src\ToDo.UI\App.xaml.cs	17	Active
Uno.Wasm.Bootstrap and Uno.Wasm.Bootstrap.DevServer are 3.3.1

**Reply by jeromelaban** [reply]
Thanks. Can you provide an actual build error? (The messages above are from the error window)
